static/src/test/components/CheckboxCreator.spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-undef */
```

Comment:
you can get rid of these directives if you add [eslint-plugin-jest](https://github.com/bloomberg/pybossa-default-theme/pull/86/files#diff-d2eb6fa44a5fa55976b7240cd7689a14R43)

Reply:
cool will do.